python/lsst/daf/butler/remote_butler/server_models.py

```diff
@@ -64,4 +64,17 @@ class GetFileByDataIdRequestModel(pydantic.BaseModel):
     collections: CollectionList
 
 
-GetFileResponseModel = FileDatastoreGetPayload
+class GetFileResponseModel(pydantic.BaseModel):
+    """Response model for get_file and get_file_by_data_id."""
+
+    dataset_ref: SerializedDatasetRef
+    artifact: FileDatastoreGetPayload | None
```

Comment:
This change breaks compatibility with previous versions of client code. Discussed with Russ and we are just going to bump the Butler library versions for datalinker/vo-cutouts and update Phalanx simultaneously with the new version of Butler server.